mscore/inspector/inspectorBase.cpp

```diff
@@ -531,7 +531,7 @@ void InspectorBase::mapSignals(const std::vector<InspectorItem>& il, const std::
                         QMenu* menu = new QMenu(this);
                         resetButton->setMenu(menu);
                         resetButton->setPopupMode(QToolButton::MenuButtonPopup);
-                        QAction* a = menu->addAction(tr("set style"));
+                        QAction* a = menu->addAction(tr("Set style"));
```

Comment:
If that indeed is what that button is all about. And if it it, it doesn't really work, see https://musescore.org/en/node/151491#comment-619726